docs/application/dotnet/samples/watch/overview.md

```diff
@@ -88,6 +88,21 @@ for TD:
 			</ul></p>
 			</td>
 		</tr>
+		<tr>
+			<td><img alt="" height="180" src="media/w42watchfacesdc.png" width="180"/></td>
+			<td>
+                        <p><strong>(W) WatchFaceSDC</strong> [In progress]</p>
+			<p>This sample application demonstrates how to create watch face app using Xamarin Forms.</p>
+                        <p>Additionally, it presents how to extend functionality of the app by displaying battery related properties on the watch face, using data provided by <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.Battery.html" target="_blank">Tizen Battery API</a>.</p>
```

Comment:
Can we rephrase as:
Additionally, it shows how to extend functionality of the app by displaying battery related properties on the watch face. Tizen Battery API provides data for extending the functionality. 

Please verify whether the technical meaning is altered.

Reply:
Updated